wrims/build.gradle

```diff
@@ -0,0 +1,56 @@
+plugins{
+    id 'library.deps-conventions'
+    id 'library.java-conventions'
+
+    // The following plugin is used to build the OSGI bundle
+    id 'biz.aQute.bnd.builder' version '6.4.0'
+}
+
+repositories {
+    mavenLocal()
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
```

Comment:
This seems like it'd be global across the project? Should be in the java-conventions file referenced on line 3

Reply:
you're right, it's already in the java-conventions. Removed.